src/main/kotlin/no/nav/dagpenger/joark/mottak/JoarkMottak.kt

```diff
@@ -100,7 +101,14 @@ class JoarkMottak(val env: Environment, private val journalpostArkiv: Journalpos
 
     private fun hentInngåendeJournalpost(journalpostId: String): Behov {
         val journalpost = journalpostArkiv.hentInngåendeJournalpost(journalpostId)
-        jpCounter.inc()
+        jpCounter
+            .labels(
+                journalpost.tema,
+                journalpost.dokumentListe.first().navSkjemaId,
```

Comment:
This looks ok - but I do remember there is an issue with label names, they cannot have `null` values, they will either be `"null"` or an NPE in runtime. Either way - it is safer to have an  eg. `orElse("unknown")`

Reply:
👍